### PR TITLE
GitHub Actions

### LINKED ISSUE #7746: Move to GitHub Actions

### PR DESCRIPTION
**Pull request by brusch**
Resolves #7746

### CODE REVIEW

File: .github/ci/files/config/packages/dev/config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: ../../config.yml }
```
Comment (dvesh3) [inline .github/ci/files/config/packages/dev/config.yaml:2]:
```suggestion
    - { resource: ../../config.yaml }
```

File: .github/ci/files/config/packages/dev/routes.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ _pimcore_dev:
     resource: "@PimcoreCoreBundle/Resources/config/routing_dev.yml"
 
 _main:
-    resource: ../../routes.yaml
+    resource: ../../routes.yml
```
Comment (dvesh3) [inline .github/ci/files/config/packages/dev/routes.yaml:5]:
```suggestion
    resource: ../../routes.yaml
```

File: .github/ci/files/config/packages/test/config.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 imports:
-    - { resource: ../dev/config.yaml }
+    - { resource: ../dev/config.yml }
```
Comment (dvesh3) [inline .github/ci/files/config/packages/test/config.yaml:2]:
```suggestion
    - { resource: ../dev/config.yaml }
```

File: composer.json [hunk 4]
```diff
@@ -106,7 +106,7 @@
   "require-dev": {
     "cache/integration-tests": "dev-master",
     "codeception/codeception": "^4.1.12",
-    "codeception/module-symfony": "^1.5.0",
+    "codeception/module-symfony": "^1.6.0",
```
Comment (dvesh3) [inline composer.json:109]: since we bumped it to 1.6.x, let's remove line `"codeception/module-symfony": "1.5.1"` from conflicts 😉

Comment (brusch) [reply composer.json:108]: Good catch, thanks a lot!

**Pull request merged**